csi-driver-smoke.yml

```diff
@@ -0,0 +1,44 @@
+- job:
+    name: gluster_csi-driver-smoke
+    node: gluster
+    description: Smoke test for gluster-csi-driver
+    project-type: freestyle
+    concurrent: true
+
+    scm:
+    - git:
+        url: https://github.com/gluster/centosci.git
+        branches:
+        - origin/master
+
+    properties:
+    - github:
+        url: https://github.com/gluster/gluster-csi-driver/
+
+    triggers:
+    - github-pull-request:
+        admin-list:
+        - humblec
+        - Madhu-1
+        white-list:
+        - humblec
+        - Madhu-1
+        cron: H/5 * * * *
+        status-context: centos-ci
+
+    builders:
+    - shell: !include-raw: scripts/common/get-node.sh
+    - shell: scripts/common/bootstrap.sh $WORKSPACE/scripts/csi-driver-smoke/run-test.sh "ghprbPullId=${ghprbPullId}"
```

Comment:
I don't like the fact that the script to run those test comes from out of this repo. This is not clean. It should be under the control of the people working on the csi-driver itself. See for instance the heketi integration. The ci runner scripts reside in their own repo https://github.com/heketi/ci-tests which is referenced in the ci-job: https://github.com/gluster/centosci/blob/master/heketi-functional.yml#L24 . The scripts referenced to run and bootstrap the jobs are in that heketi repo. So the devs of the project to be tested do not always need to beg for some time of the admins of this repo when they need to change how tests are run or which...